odoo/addons/base/models/res_partner.py

```diff
@@ -170,7 +170,7 @@ def _split_street_with_params(self, street_raw, street_format):
     category_id = fields.Many2many('res.partner.category', column1='partner_id',
                                     column2='category_id', string='Tags', default=_default_category)
     credit_limit = fields.Float(string='Credit Limit')
-    barcode = fields.Char(oldname='ean13', help="Use a barcode to identify this contact from the Point of Sale.")
+    barcode = fields.Char(oldname='ean13', help="Use a barcode to identify this contact from the Point of Sale.", copy=False)
```

Comment:
This field should be assigned in the PoS module. Same goes for the view. I did a quick check in the code and the field is not used outside of the PoS.